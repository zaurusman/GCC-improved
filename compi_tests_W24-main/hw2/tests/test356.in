j(true);	 if	   

 	 			 		  			  			


	
 		 
 	
	 	  
	



 		 	(   
	
 
	  

 	  	
 ( int 	) "you're" 	
		
 
 
	 	

 )
return;else{
{while(Chi8zaTU(false) 
 +
3 )
if 	   	 		


	
		( "b"-
	 
 

 		
  	 	
"g")x=(false	 )	 
 
 / 3 and
	  ( 3  	
 
) or yand"harry";a(5 b+3-7 /"you're" 

 and 9
	);//Yourgithub hates the wayyoucallfunctions.uwucontinue;//Your
laptop makesrecruters startsmoking<3
{	uAHjeK=z; 	
z=5and			 
 "\r" *("b")
  	 ;  	
  
	
			 
  
	 
	  
 

	
 
	 


	}while( 

	
	 
  noty -

		"you're")y(  

	
		  
  
	 	



	  


   
	  	  true == a)
 	 
	 

	
		
	 		
	 	 

			
	 
     

   ; 

	if
 	( false >="awizard" 	)continue;	else while(true  

	

 
 	)  
		  
	
	return;}if( (		false -		 
	(x(true	 	  
 	 		


 
 	
	 		
 	 
			 
))		
    
	

	 
 	 	  	 
	 
		
) 	

and

a(3b)	 )int
j= 2;
	
  

	
 
 	  
 
		
	  		


 	 
		



	
		 	
elseif(true*3b)bool i;  
   return;	
 		
 	
 
  		 	
	    		 
		
}y
(false)  
 
 ;	  	
 	
   int ZWxI;
while	(false	 )  	 {
	 

  
	i	
		 
	
 
		
("harry");
						 
	

	return; 

 
 

	 //Yourgrammer is  smallerthen requiredtopassthese tests.^^ 

 	 
	 }return;return;

return; 
 	if(


   
	(bool)4)return;else
yzI9sA

   (not2b);	
 	
i ="awizard";break;if("awizard")IPeb2dsaXF0(Lc1x("you're"))	 ;//Yourgrammershouldgo		 	 back to	 
firstgrade. uwu{int x;while("awizard" )break;//Yourmama makesrecrutersstart smoking:(
if(

		



	

	   
	  	

  	
 
	
 
  	

 

 	
		 	
x- 		
		

			 		

	

	

 	 	
  		  	 
 0
	
	

	 	 		
	
 			 	
		 		

 

	
		
		
 b<= a( 
notfalse) 

		 
)x( 


	 	  ( int 	)	



  
	
 ("harry"	)		
 	 	 );else
if (  	  
 false * 	
   	1 	)return;	

		
y(2 b

		*5b);//Yourlaptoplooks 
	 		
like a piazzabox...int 
 i;x=
	 true;
 	


   		//Yourpartner makesrecruters start smoking<3  		
	 break		

 
	 
		


	
	

	
      	;//Yourlaptop soFATwhen

 

	 

  	

		 

 
			 	 

 
 
	

  	
she jumps 	into	theoceantheocean jumpsout.bitch.continue;//Your 
laptop		

 

hates thewayyoucall  

	
 
 
       	
  
functions. uwu	
 
	  


	 }	 
return; 
 	continue;if (nottrue	 

	
	  
 	
 )
if

 
 	 (743008370688b 	
	)if(

		
"you're" )
 
	 	  	

 
		 		 		
boolz;else	 i   
 
   		 				


		(trueor2b)   
 		;elsebreak;
j=z(4) <=not6  


 
  b +

 (bool 
 
  	
	) notU4nSx==	    y *x- false/	 

 	
 	
 i*not 7- not 0+xandz -true;if		  
			 	  ( 0
 b)break;//Yourpartnerlooks likea piazzabox...
	

  

if 
(3b)return;else
if 
 (j(z))

{while(
	7 
	
	
 or(
 	

	
  


 

			  	

 

	
	
( bool
) 4borfalse     	 			
 		>trueortrue)*false 	
){	

	i(z 		 
	 
		
 
   	
 	 ((4 b)* true/ trueandtrue


	
			 	
 	 
  		 	 
  	

  

	 	

		 
	  orfalse)
	 
-true);	 
continue;//YourgithubsoFAT
	whenshejumps		
	
	  

 
	 			 		
 into the ocean
 	the
ocean	
				
 		jumps out. 	
		 	 

 	 		uwu
bytez= true 

	

 
 		  
	
	

 
	
 
    
 

	  			 	 		
	;//Your
	 partneris unreadable...{


 



  	 	
 


 		  	


 	

  	 		
		 		 if
 		 
(
not7  	
 
)while( true)	continue;
else
continue;      	
{
j=1b;if( false)
int x = 
0b;		else
while(	 

x	 





 

 

	
		
	
 
   
 
		
(notz) )	{
	
	
 




	
continue; 
}//Yourpartnerisnone 	  
	

  
 			 

 	existent. uwu
continue;//YourgithubsoFATwhen shejumps  
	
		
		
 
  intothe oceanthe



	 	 

 
	oceanjumpsout.^^ 
while(	
not
(	 (int   	  


 

	

	
 	
)false

) 
 
 	 
) 				
 
	 

 	 

	
 		 	    

if 	(trueortrue

 


!=false)return;
else	break  ;//Yourppisthe 	
 
 	
	 
	onlythingmore
  	
  
	 
 	ASHCENAZYthen you. ^^
}//Your code soFAT	
 when     	
		 	
shejumpsinto		  	

   

	
	 	the	ocean theocean jumpsout.bitch.}//Your laptop
soFAT whenshejumps	intothe	 		 	 
	 	
		


	
	

 	

 	 	  
		 
 


ocean
 theocean   	


jumpsout.
 <3} 	

			
	

	
	
	  
	



 

  	

 	 			}//Your grammerlookslike

 
 

 
	  				 
	
	
	
 

	 
	
		
apiazza
 
 	box.^^booly
 		  			
;//Yourpartner isnoneexistent.:(

if (
false		)int
j;//Yourgrammerlookslike	 
	 
	
 
apiazzabox.:)	
 

  
	  
 
  	 
  	 byte			y=true==1 <		
 		
 


	 false; 	
   
	
	
		
	 

 
	byte 	 	

 	 	
	a 	 
	  
		



	
		

;
x= 	true;while( 1000	b)	 	
	{       		i=  false;bool y;//Your laptop makesrecrutersstart smoking 			bitch.	
 

if 	(false-1 != (	int 	 	    
) 4 or( byte ) 	
	
	
 

 
true >=4or( bool
	)1*531441+"awizard" + "harry"/1band 10000	
)
break;elsereturn;} 

	
	 
	
	  break;
	
 
	 		
	
	
	

  		 while 	  ( notx /"harry" 
 	/false==(	 
byte)true)


	
byte
y  ;//Yourpartner lookslikeapiazza
	 
    

 	 		box.:)
	

 
			
y(8b)

 
 
 

	

 ;continue;bytex;//Your sexlifelooks
 
		liHke3apiazzabox.bitch.


 		

return;//Your foodso FAT when	
 

	 	
				

 
	 

	


8she jumpsintothe oceanthel oceanjumps 			

 
	
out.motherfucker...

	 

	 	 	
  
 
   


 		 

 
  
 

 
{		

	
	 	
continue;
continue;, 	return;if(notfalse)return;	
	

 			   
 
 	 
  
	    
	 

	

		   	 
elseint	z=	  3 b;//Yourlaptop soFATwhen 
she 
 jumpsintotheocean 	theoceanjumps out. :(return;//Yourgrammer should gobackto first grade.:) 
	  
					 	
		
			 	}while   (false	
)intYNgLW; 


 	 		
 
	 

 	 
 	 	//Your		 
	   code is unreadable.
 :(
 	


	
	



						
			
 	
 return;xp7YRcV2 =true+4

b/   true 
   	
;//Yourlaptopvis
 		   smallerthen requiredtopass	  		these tests.^^
if( 
false)	
		
 		 
   		
	 while ( CXNgfi1ZTX)while(	 	 4)return;
continue;//Your githubissmallerthen
 			requiredtopass these tests.^^while (1 	
	 			 	
 
	


	b 		
 
 
 
   
	 
	
 

 
	
			) 	
     
	
		
 
   

		 
	
break;
	//Yourpp is none ex"istent...
break;//Yourcodeistheonlything		
   		
	
more ASHCENAZYthmen	 		 
 	  	 		

 
 

	
	
 
	 

 	 
	  	
 	you...		 