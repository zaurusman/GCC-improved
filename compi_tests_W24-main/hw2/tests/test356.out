26: Exp -> TRUE
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
2: Statements -> Statement
16: Type -> INT
25: Exp -> STRING
32: Exp -> LPAREN Type RPAREN Exp
9: Statement -> RETURN SC
27: Exp -> FALSE
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
25: Exp -> STRING
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
27: Exp -> FALSE
19: Exp -> LPAREN Exp RPAREN
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
19: Exp -> LPAREN Exp RPAREN
29: Exp -> Exp AND Exp
21: Exp -> ID
30: Exp -> Exp OR Exp
line 46: syntax error
